book/src/troubleshooting.md

```
@@ -25,22 +25,22 @@ kubectl -n collectors logs -f daemonset/logfwd
 
 | Symptom | First check | Expected output | If not expected |
 |---|---|---|---|
-| No logs arrive at destination | `curl -s http://localhost:9090/admin/v1/status | jq '.pipelines[0].inputs'` | `lines_total` increasing | Fix file path/mount permissions |
-| Logs read, but nothing forwarded | `curl -s http://localhost:9090/admin/v1/status | jq '.pipelines[0].transform'` | `lines_in > 0` and `lines_out > 0` | Transform filter dropping all rows |
+| No logs arrive at destination | `curl -s http://localhost:9090/admin/v1/status | jq '.pipelines | .[0].inputs // (to_entries[0].value.inputs)'` | `lines_total` increasing | Fix file path/mount permissions |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)